_posts/2016-10-07-ohio-linuxfest.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: default
+title: Ohio LinuxFest
+categories: events
+tags: events
+expire_date: 2016-10-09
+---
+
+<!-- INSERT TEXT HERE -->
+Come join us today and tomorrow to enjoy Ohio LinuxFest. It's a great opportunity to both learn about Open Source Software and meet other members of the Open Source community. Open Source club will be there at the Open SOurce Club booth around 1600EDT and we'd love to see you guys stop by to say hello!
```

Comment:
SOurce -> source, but I'm merging this anyway and fixing it on my end for time reasons